weed/pb/master.proto

```diff
@@ -70,7 +70,7 @@ message Heartbeat {
 
   map<string, uint32> max_volume_counts = 4;
   uint32 grpc_port = 20;
-
+  repeated string LocationUUIDs = 21;
```

Comment:
please follow the naming convention here.

If a concern, you can use uuid as one word instead of all upper cased UUID.

Reply:
Done